cmd/main.go

```diff
@@ -33,9 +33,10 @@ func init() {
 }
 
 var (
-	endpoint          = flag.String("endpoint", "unix:/tmp/csi.sock", "CSI endpoint")
-	gceConfigFilePath = flag.String("cloud-config", "", "Path to GCE cloud provider config")
-	vendorVersion     string
+	endpoint            = flag.String("endpoint", "unix:/tmp/csi.sock", "CSI endpoint")
+	cloudConfigFilePath = flag.String("cloud-config", "", "Path to GCE cloud provider config")
+	runAsController     = flag.Bool("run-as-controller", false, "If set to true then the CSI driver runs the controller only and does not start the node service.")
```

Comment:
don't want to expand the scope of this PR too much but thinking about the "end-state" of splitting this out is useful to inform the design.

IMO the driver should have 3 distinct booleans:
`run-controller-service`, `run-node-service`, (maybe) `run-identity-service`.

These should all default to true. You can turn each of them off piecemeal if required. WDYT?